src/hooks/use-run-tsx/index.tsx

```diff
@@ -64,21 +64,27 @@ export const useRunTsx = ({
       const preSuppliedImports: Record<string, any> =
         preSuppliedImportsRef.current
 
-      const __tscircuit_require = (name: string) => {
+      for (const [importName, importValue] of Object.entries(
+        userImports ?? {},
+      )) {
+        preSuppliedImports[importName] = importValue
+      }
+
+      const tscircuitRequire = (name: string) => {
         if (!preSuppliedImports[name]) {
           throw new Error(
             `Import "${name}" not found (imports available: ${Object.keys(preSuppliedImports).join(",")})`,
           )
         }
         return preSuppliedImports[name]
       }
-      ;(globalThis as any).__tscircuit_require = __tscircuit_require
+      ;(globalThis as any).__tscircuit_require = tscircuitRequire
```

Comment:
we should revert this, variable transparency (generally changing the name of a variable that represents the same thing should be avoided)

Reply:
I think biome did this, will update